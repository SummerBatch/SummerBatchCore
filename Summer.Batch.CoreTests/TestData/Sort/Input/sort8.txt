---111BCD-------------0000000080G0000000000E0000000000A0000000006B
---111ABC-------------0000000000C0000000000C0000000000A0000000098B
---555BCD-------------0000000080G0000000000F0000000000A0000000095B
---111BCD-------------0000000080G0000000000G0000000000A0000000022B
---111ABC-------------0000000045G0000000000B0000000000A0000000005B
---111BCD-------------0000000080G0000000000H0000000000A0000000001B
---555BCD-------------0000000080G0000000000I0000000000A0000000000B
---111ABC-------------0000000001B0000000000A0000000000A0000000071B
---555ABC-------------0000000080G0000000000D0000000000A0000000000B