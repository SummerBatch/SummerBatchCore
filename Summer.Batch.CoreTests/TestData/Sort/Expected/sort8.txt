---111ABC-------------0000000047B0000000000F0000000000C0000000174F
---111BCD-------------0000000242A0000000002{0000000000C0000000029F
